exatomic/core/atom.py

```diff
@@ -349,18 +350,68 @@ class Frequency(DataFrame):
     +-------------------+----------+-------------------------------------------+
     | dz                | float    | atomic displacement in z direction (req.) |
     +-------------------+----------+-------------------------------------------+
+    | ir_int            | float    | ir intensity of the vibrational mode      |
+    +-------------------+----------+-------------------------------------------+
     | symbol            | str      | atomic symbol (req.)                      |
     +-------------------+----------+-------------------------------------------+
     | label             | int      | atomic identifier                         |
     +-------------------+----------+-------------------------------------------+
     """
+    _index = 'frequency'
+    _cardinal = ('frame', np.int64)
+    _categories = {'symbol': str,
+                   'label': np.int64, 'frequency': np.float64}
+    _columns = ['dx', 'dy', 'dz', 'symbol', 'frequency', 'freqdx', 'ir_int']
     #@property
     #def _constructor(self):
     #    return Frequency
 
     def displacement(self, freqdx):
         return self[self['freqdx'] == freqdx][['dx', 'dy', 'dz', 'symbol']]
 
+    def ir_spectra(self, fwhm=15, lineshape='gaussian', **kwargs):
```

Comment:
I think we could write tests for this function that essentially patch (or perhaps mock) the functions being used in plotting but still test whether the boolean logic is correctly handled and the ranges are created correctly and such.

Reply:
I'll look into that. I wasn't quite sure how to write a test for a visualization method.